src/components/List/ListItem.tsx

```diff
@@ -67,29 +72,35 @@ class ListItem extends UIComponent<Extendable<ListItemProps>, ListItemState> {
     important: PropTypes.bool,
     media: PropTypes.any,
 
-    selection: PropTypes.bool,
+    selectable: PropTypes.bool,
+    selected: PropTypes.bool,
+
     truncateContent: PropTypes.bool,
     truncateHeader: PropTypes.bool,
 
     accessibility: PropTypes.func,
+    onClick: PropTypes.func,
   }
 
   static defaultProps = {
     as: 'li',
     accessibility: listItemBehavior as Accessibility,
   }
 
-  constructor(props: ListItemProps) {
-    super(props, null)
+  private itemRef = React.createRef<HTMLElement>()
```

Comment:
where this guy is used? seems that we should delete it

Reply:
I don't know this guy either, will delete it